core/src/main/scala/com.mediative.amadou/SparkImplicits.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017 Mediative
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mediative.amadou
+
+import org.apache.spark.sql._
+import org.apache.spark.sql.functions.when
+
+object SparkImplicits {
+
+  /**
+   * Helpers for working with Dataset columns.
+   */
+  implicit class SparkColumnOps(val self: Column) extends AnyVal {
+
+    /**
+     * Filter out values which are null, empty or "null".
+     *
+     * @example
+     * {{{
+     * dataset.filter($"UserId".isDefined)
+     * }}}
+     */
+    def isDefined: Column =
```

Comment:
`isXXX` methods traditionally return a `Boolean`, this should be named something like `filterDefined`.